src/db/migrations/0081_confused_gwen_stacy.sql

```
@@ -0,0 +1,17 @@
+--> statement-breakpoint
+-- Social providers that declare a real OIDC issuer in better-auth.
+UPDATE "account" SET "issuer" = 'https://accounts.google.com' WHERE "provider_id" = 'google' AND "issuer" IS NULL;
+--> statement-breakpoint
+UPDATE "account" SET "issuer" = 'https://appleid.apple.com' WHERE "provider_id" = 'apple' AND "issuer" IS NULL;
+--> statement-breakpoint
+UPDATE "account" SET "issuer" = 'https://www.facebook.com' WHERE "provider_id" = 'facebook' AND "issuer" IS NULL;
+--> statement-breakpoint
+UPDATE "account" SET "issuer" = 'https://access.line.me' WHERE "provider_id" = 'line' AND "issuer" IS NULL;
+--> statement-breakpoint
+-- Remaining OAuth providers without a declared issuer use the synthetic OAuth issuer.
+UPDATE "account" SET "issuer" = 'local:oauth:' || "provider_id" WHERE "issuer" IS NULL;
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)